### DISCUSSION TITLE
Show "favorites" countries on top of list

### DISCUSSION THREAD

**Discussion by davoypres** [AstrOOnauta/react-native-international-phone-number#41] [Q&A] [answered]
Hello! This is a great lib..thanks for that!!!!

Is there any way to mark countries at the top of the list? If not, please,  add some feat to add some countries to the top of the countries list to make it easier to choose.

**Comment by AstrOOnauta** [comment]
Hey @davoypres ... not for now, but I will soon!!

**Comment by AstrOOnauta** [comment, marked as answer]
@davoypres I released a next version with new feat `add some countries to the top of the countries list`.
Please update the lib version to v0.7.1 and use the following code to solve your question:
```
...
  <PhoneInput
    ...
    popularCountriess={['BR', 'PT', 'CA', 'US']}  <-- Add this props
  />
  ...
```

OBS.: Referenced in this new point in the docs:
https://github.com/AstrOOnauta/react-native-international-phone-number#show-popular-countries-at-the-top-of-the-countries-list-inside-modal

🚀AstrOOnauta 🚀

**Reply by davoypres** [reply]
Thanks for you fast reply!

This worked like a charm ❤️
